common/src/universal/run-tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -ex
+
+trap "docker stop $IMAGE_NAME; exit 1" EXIT
+
+docker run -d \
+  -e "discovery.type=single-node" \
+  -e "traveltime.app.id=id" \
+  -e "traveltime.api.key=key" \
+  -e "xpack.security.enabled=false" \
+  --rm \
+  --name $IMAGE_NAME \
+  $IMAGE_NAME
+
+docker exec $IMAGE_NAME ./wait-for-startup.sh
```

Comment:
the path for these shell scripts seems to be incorrect

Reply:
These path are paths inside the container relative to its working directory. When building the image I call `COPY path/to/script ./`, where `./` is the working directory of the container.